test/branded_token/accept_stake_request.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const { AccountProvider } = require('../test_lib/utils.js');
+const { Event } = require('../test_lib/event_decoder.js');
+
+const web3 = require('../test_lib/web3.js');
+const utils = require('../test_lib/utils');
+const brandedTokenUtils = require('./utils');
+
+contract('TestBrandedToken::requestStake', async () => {
+    contract('Event', async (accounts) => {
+        const accountProvider = new AccountProvider(accounts);
+
+        it('Checks that StakeRequestAccepted & Transfer events are emitted successfully.', async () => {
```

Comment:
`it` captions the test case (i.e., the functionality being tested), not the test. In other words. Ex. from Mocha docs:
```
describe('Array', function() {
  describe('#indexOf()', function() {
    it('should return -1 when the value is not present', function() {
```

"Checks that StakeRequestAccepted & Transfer events are emitted successfully." ->

"StakeRequestAccepted & Transfer events are emitted successfully" or (my preference for using fewer words) "Emits StakeRequestAccepted and Transfer events".

Reply:
I think I am not completely clear here:
We discussed in previous PRs "It should make a proper statement". Even the mocha it statement makes proper it statement.

Below doesn't make proper it statement 
it "StakeRequestAccepted & Transfer events are emitted successfully"

Below is perfect: 
it "Emits StakeRequestAccepted and Transfer events" 👍

Reply:
I understood your point of "captions the test case"